### ISSUE TITLE
SelectField default value display issue

### ISSUE THREAD

**Issue by Sir4ur0n** [erikras/redux-form-material-ui#25]
Hi,

When testing the SelectField example, I don't get the same display.
```javascript
<Field
            name="driver"
            component={SelectField}
            hintText="Driver"
            floatingLabelText="Driver">
            <MenuItem value="alice@redux-pizza.com" primaryText="Alice"/>
            <MenuItem value="bob@redux-pizza.com" primaryText="Bob"/>
            <MenuItem value="carl@redux-pizza.com" primaryText="Carl"/>
</Field>
```
The first option's primaryText and the floatingLabelText overlap:
![image](https://cloud.githubusercontent.com/assets/1204125/17486769/63855e36-5d92-11e6-873d-4f27af8044ab.png)

Using
"redux-form": "6.0.0-rc.4",
"redux-form-material-ui": "4.0.1"

Am I the only one?

**Comment by erikras** [comment]
Hmm... I'm not seeing that on [the live demo](http://erikras.github.io/redux-form-material-ui/). You are seeing it when running the example locally in development mode?

**Comment by itaylorweb** [comment]
I get the same.
Looks like an issue with Material-UI 0.15.3.  See the Material-UI Select Field Floating label example: http://www.material-ui.com/#/components/select-field

If you want a value to be selected you could initialize the form with just the driver field which appears to correct the floating label.

**Comment by erikras** [comment]
Okay, so it sounds like the problem is for Material UI to solve.

Thanks, @itaylorweb.

**Comment by Sir4ur0n** [comment]
You are right @itaylorweb it seems related to https://github.com/callemall/material-ui/issues/2446

**Comment by m2mathew** [comment]
This issue is fixed in MUI v0.15.4, released yesterday.
https://github.com/callemall/material-ui/releases/tag/v0.15.4

**Comment by course-js** [comment]
Sucks ....
Sorry guys but ...
It's probably only one way to make form of editing.
It cannot be restored from initial redux Form values.
```
export default function ({ input, label, meta: { touched, error }, onChange, children, ...custom }) {
    if ( input.value === '' && custom.cvalue ) { // hack for f....g redux form with f.....g material components
        input.value = String(custom.cvalue);
    }
    return (
        <SelectField
            {...input}
            {...custom}
            fullWidth={true}
            children={children}
            floatingLabelText={label}
            errorText={touched && error}
            onChange={(event, index, value) => {
                if ( is.function(onChange) ) { // and custom onChange for f....g ....
                    value = onChange(event, index, value);
                }
                input.onChange(value);
            }}/>
    )
}
```
then in template
its can be way to make form of editing existing entity ....

```
<Field
    name="study"
    label="Studies"
    component={ FormSelect }
    cvalue={this.state.study.id}
    onChange={(event, index, value) => {
        this.setState({study: _.find(studies, {id: value})||{id: 0}});
        return value;
    }}>
    {studies.map( (study, key) => ( <MenuItem key={key} value={study.id} primaryText={study.officialTitle} /> ))}           
</Field>
```
